mgmtsystem/azure.py

```diff
@@ -73,7 +77,7 @@ def run_script(self, script):
         self.logger.info("PowerShell Returned:\n{}\n".format(result.std_out.strip()))
         return result.std_out.strip()
 
-    def start_vm(self, vm_name, resource_group):
+    def start_vm(self, vm_name, resource_group=default_resource_group):
```

Comment:
I thought about that, but didn't want to do it 17 times when a single line would work.  You know me.  I'll clean that up.  Thanks guys.